Develop

temcdrm opened the pull request:
This would be the base version for DSO+T study

* Using the FNCS message aggregation feature. If one subscriber uses aggregation, it appears that all subscribers to the same publisher must also use aggregation.
* Refactored and documented tesp_support classes
* New installers for Windows and Linux
* ERCOT and all the examples run after refactoring
* Updates will be required for the Docker version, and possibly for the ERCOT monitor 
* See docstring conventions for the Python code in tesp_support; this appears formatted on ReadTheDocs

xcosmos6 commented:
Hi Tom,

After approve the pull request, I don't need to merge it to the master branch, right? I assume you will be doing that?

Thanks,
Heng
________________________________
From: Tom McDermott <notifications@github.com>
Sent: Wednesday, February 13, 2019 2:52 PM
To: pnnl/tesp
Cc: Subscribed
Subject: [pnnl/tesp] Develop (#14)


This would be the base version for DSO+T study

  *   Using the FNCS message aggregation feature. If one subscriber uses aggregation, it appears that all subscribers to the same publisher must also use aggregation.
  *   Refactored and documented tesp_support classes
  *   New installers for Windows and Linux
  *   ERCOT and all the examples run after refactoring
  *   Updates will be required for the Docker version, and possibly for the ERCOT monitor
  *   See docstring conventions for the Python code in tesp_support; this appears formatted on ReadTheDocs

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/pnnl/tesp/pull/14

Commit Summary

  *   FNCS updates
  *   updates for dictionary changes and FNCS message aggregation
  *   first file with docstrings
  *   for readthedocs to work
  *   looking for tesp_support
  *   Python 3.6 for ReadTheDocs
  *   Python 3.6 for ReadTheDocs
  *   class refactoring
  *   updating module selections
  *   working on fix to the tesp_monitor for te30
  *   testing with refactored classes
  *   refactored precooling classes
  *   precool agent estimates the ETP parameters
  *   follow TE30 rules for the inverter meters
  *   testing the sgip1 case; all simulations run but the HVAC rebound effect at 6 p.m. each day is more pronounced than before:
  *   fixing tesp_monitor
  *   reorganized the sections, added the Napoleon extension for Python docstrings
  *   autonumber figures and tables
  *   rolling the version number
  *   testing on Windows
  *   rebuild for possible fncs aggregation
  *   rebuild for FNCS message aggregation
  *   testing on Linux
  *   some docstrings
  *   installer for 0.3.0
  *   updating documentation
  *   adding docstrings
  *   docstrings
  *   docstrings for simple_auction, added Todo lists
  *   finished the docstrings
  *   rolling version number with docstrings
  *   updates to kill5570 for Windows 7, and python3 added to some Linux/Mac shell scripts
  *   installation tested for Ubuntu 18.04 LTS
  *   updating instructions
  *   updating example instructions
  *   draft documentation for DSO+T agents
  *   updating installation and build instructions
  *   added exterior doors to GridLAB-D dictionary

File Changes

  *   M README.md<https://github.com/pnnl/tesp/pull/14/files#diff-0> (28)
  *   A doc/Bibliography.rst<https://github.com/pnnl/tesp/pull/14/files#diff-1> (96)
  *   A doc/Building_TESP.rst<https://github.com/pnnl/tesp/pull/14/files#diff-2> (26)
  *   R doc/Code_Reference.rst<https://github.com/pnnl/tesp/pull/14/files#diff-3> (52)
  *   A doc/DSOT_Study.rst<https://github.com/pnnl/tesp/pull/14/files#diff-4> (63)
  *   A doc/Design_Reference.rst<https://github.com/pnnl/tesp/pull/14/files#diff-5> (574)
  *   A doc/Installing_MacOSX_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-6> (2)
  *   A doc/Installing_Ubuntu_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-7> (2)
  *   A doc/Installing_Windows_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-8> (2)
  *   A doc/Linux_Build_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-9> (1)
  *   A doc/MacOSX_Build_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-10> (1)
  *   A doc/Next_Version.rst<https://github.com/pnnl/tesp/pull/14/files#diff-11> (214)
  *   A doc/Running_Examples_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-12> (2)
  *   A doc/SGIP1_Example.rst<https://github.com/pnnl/tesp/pull/14/files#diff-13> (315)
  *   M doc/TESP.eap<https://github.com/pnnl/tesp/pull/14/files#diff-14> (0)
  *   D doc/TESP_DesignDoc.html<https://github.com/pnnl/tesp/pull/14/files#diff-15> (393)
  *   D doc/TESP_DesignDoc.rst<https://github.com/pnnl/tesp/pull/14/files#diff-16> (1376)
  *   A doc/TESP_Overview.rst<https://github.com/pnnl/tesp/pull/14/files#diff-17> (170)
  *   A doc/Using_TESP.rst<https://github.com/pnnl/tesp/pull/14/files#diff-18> (33)
  *   A doc/Windows_Build_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-19> (1)
  *   M doc/conf.py<https://github.com/pnnl/tesp/pull/14/files#diff-20> (22)
  *   M doc/index.rst<https://github.com/pnnl/tesp/pull/14/files#diff-21> (13)
  *   M doc/media/GLDDictionary.png<https://github.com/pnnl/tesp/pull/14/files#diff-22> (0)
  *   M doc/media/TESPComposition.png<https://github.com/pnnl/tesp/pull/14/files#diff-23> (0)
  *   M doc/media/tesp_support.png<https://github.com/pnnl/tesp/pull/14/files#diff-24> (0)
  *   A doc/requirements.txt<https://github.com/pnnl/tesp/pull/14/files#diff-25> (5)
  *   M doc/tesp_support.inc<https://github.com/pnnl/tesp/pull/14/files#diff-26> (66)
  *   M ercot/case8/.gitignore<https://github.com/pnnl/tesp/pull/14/files#diff-27> (1)
  *   M ercot/case8/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-28> (1)
  *   M ercot/case8/plot_gld.py<https://github.com/pnnl/tesp/pull/14/files#diff-29> (4)
  *   R ercot/case8/prep_ercot_substation.py<https://github.com/pnnl/tesp/pull/14/files#diff-30> (14)
  *   M ercot/case8/prepare_case.py<https://github.com/pnnl/tesp/pull/14/files#diff-31> (18)
  *   M ercot/case8/pypower8.yaml<https://github.com/pnnl/tesp/pull/14/files#diff-32> (80)
  *   M ercot/case8/run_market.sh<https://github.com/pnnl/tesp/pull/14/files#diff-33> (16)
  *   M ercot/case8/run_market_debug.sh<https://github.com/pnnl/tesp/pull/14/files#diff-34> (16)
  *   A ercot/pdf/DSO+T_Water_Heater_Agent.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-35> (0)
  *   A ercot/pdf/DayAheadBidsBattery.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-36> (0)
  *   A ercot/pdf/EWH_11_6_2018.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-37> (0)
  *   A ercot/pdf/Fixed_Layers_Stratified_Water_Heater.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-38> (0)
  *   A ercot/pdf/MasterScriptDocumentation.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-39> (0)
  *   A ercot/pdf/Transformer_transactive_control.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-40> (0)
  *   A ercot/pdf/Water_Heater_Model_Deficiency_for_B2G.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-41> (0)
  *   M examples/Running_Examples.rst<https://github.com/pnnl/tesp/pull/14/files#diff-42> (111)
  *   M examples/energyplus/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-43> (10)
  *   M examples/energyplus/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-44> (1)
  *   M examples/ieee8500/PNNLteam/bill.py<https://github.com/pnnl/tesp/pull/14/files#diff-45> (6)
  *   M examples/ieee8500/PNNLteam/inv30.glm<https://github.com/pnnl/tesp/pull/14/files#diff-46> (27)
  *   M examples/ieee8500/PNNLteam/inv8500.glm<https://github.com/pnnl/tesp/pull/14/files#diff-47> (2)
  *   A examples/ieee8500/PNNLteam/invti30.glm<https://github.com/pnnl/tesp/pull/14/files#diff-48> (3558)
  *   M examples/ieee8500/PNNLteam/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-49> (1)
  *   M examples/ieee8500/PNNLteam/plot_invs.py<https://github.com/pnnl/tesp/pull/14/files#diff-50> (6)
  *   M examples/ieee8500/PNNLteam/prepare_cases.py<https://github.com/pnnl/tesp/pull/14/files#diff-51> (4)
  *   M examples/ieee8500/PNNLteam/run30.sh<https://github.com/pnnl/tesp/pull/14/files#diff-52> (2)
  *   M examples/ieee8500/PNNLteam/run8500.sh<https://github.com/pnnl/tesp/pull/14/files#diff-53> (2)
  *   A examples/ieee8500/PNNLteam/runti30.sh<https://github.com/pnnl/tesp/pull/14/files#diff-54> (6)
  *   M examples/ieee8500/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-55> (61)
  *   M examples/ieee8500/clean.bat<https://github.com/pnnl/tesp/pull/14/files#diff-56> (1)
  *   M examples/ieee8500/clean.sh<https://github.com/pnnl/tesp/pull/14/files#diff-57> (1)
  *   M examples/loadshed/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-58> (4)
  *   M examples/loadshed/java/JNIfncs.dll<https://github.com/pnnl/tesp/pull/14/files#diff-59> (0)
  *   M examples/loadshed/java/libJNIfncs.jnilib<https://github.com/pnnl/tesp/pull/14/files#diff-60> (0)
  *   M examples/loadshed/java/libJNIfncs.so<https://github.com/pnnl/tesp/pull/14/files#diff-61> (0)
  *   M examples/loadshed/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-62> (1)
  *   M examples/loadshed/run.sh<https://github.com/pnnl/tesp/pull/14/files#diff-63> (2)
  *   M examples/pypower/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-64> (7)
  *   M examples/pypower/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-65> (1)
  *   M examples/pypower/runpp.sh<https://github.com/pnnl/tesp/pull/14/files#diff-66> (2)
  *   M examples/sgip1/.gitignore<https://github.com/pnnl/tesp/pull/14/files#diff-67> (2)
  *   M examples/sgip1/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-68> (16)
  *   M examples/sgip1/SGIP1a.glm<https://github.com/pnnl/tesp/pull/14/files#diff-69> (2)
  *   M examples/sgip1/SGIP1b.glm<https://github.com/pnnl/tesp/pull/14/files#diff-70> (2)
  *   M examples/sgip1/SGIP1c.glm<https://github.com/pnnl/tesp/pull/14/files#diff-71> (2)
  *   M examples/sgip1/SGIP1d.glm<https://github.com/pnnl/tesp/pull/14/files#diff-72> (2)
  *   M examples/sgip1/SGIP1e.glm<https://github.com/pnnl/tesp/pull/14/files#diff-73> (2)
  *   M examples/sgip1/prepare_cases.py<https://github.com/pnnl/tesp/pull/14/files#diff-74> (20)
  *   M examples/sgip1/pypower.yaml<https://github.com/pnnl/tesp/pull/14/files#diff-75> (10)
  *   M examples/sgip1/runSGIP1a.bat<https://github.com/pnnl/tesp/pull/14/files#diff-76> (4)
  *   M examples/sgip1/runSGIP1a.sh<https://github.com/pnnl/tesp/pull/14/files#diff-77> (4)
  *   M examples/sgip1/runSGIP1b.bat<https://github.com/pnnl/tesp/pull/14/files#diff-78> (4)
  *   M examples/sgip1/runSGIP1b.sh<https://github.com/pnnl/tesp/pull/14/files#diff-79> (4)
  *   M examples/sgip1/runSGIP1c.bat<https://github.com/pnnl/tesp/pull/14/files#diff-80> (4)
  *   M examples/sgip1/runSGIP1c.sh<https://github.com/pnnl/tesp/pull/14/files#diff-81> (4)
  *   M examples/sgip1/runSGIP1d.bat<https://github.com/pnnl/tesp/pull/14/files#diff-82> (4)
  *   M examples/sgip1/runSGIP1d.sh<https://github.com/pnnl/tesp/pull/14/files#diff-83> (4)
  *   M examples/sgip1/runSGIP1e.bat<https://github.com/pnnl/tesp/pull/14/files#diff-84> (4)
  *   M examples/sgip1/runSGIP1e.sh<https://github.com/pnnl/tesp/pull/14/files#diff-85> (4)
  *   M examples/sgip1/runSGIP1ex.bat<https://github.com/pnnl/tesp/pull/14/files#diff-86> (4)
  *   M examples/sgip1/runSGIP1ex.sh<https://github.com/pnnl/tesp/pull/14/files#diff-87> (5)
  *   M examples/sgip1/runall.sh<https://github.com/pnnl/tesp/pull/14/files#diff-88> (13)
  *   M examples/te30/.gitignore<https://github.com/pnnl/tesp/pull/14/files#diff-89> (3)
  *   M examples/te30/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-90> (26)
  *   A examples/te30/TESP_Monitor.yaml<https://github.com/pnnl/tesp/pull/14/files#diff-91> (36)
  *   M examples/te30/TE_Challenge.glm<https://github.com/pnnl/tesp/pull/14/files#diff-92> (2)
  *   M examples/te30/clean.bat<https://github.com/pnnl/tesp/pull/14/files#diff-93> (1)
  *   M examples/te30/clean.sh<https://github.com/pnnl/tesp/pull/14/files#diff-94> (2)
  *   M examples/te30/eplus_json.yaml<https://github.com/pnnl/tesp/pull/14/files#diff-95> (2)
  *   D examples/te30/fncs.zpl<https://github.com/pnnl/tesp/pull/14/files#diff-96> (23)
  *   A examples/te30/gui.bat<https://github.com/pnnl/tesp/pull/14/files#diff-97> (3)
  *   A examples/te30/gui.py<https://github.com/pnnl/tesp/pull/14/files#diff-98> (6)
  *   A examples/te30/gui.sh<https://github.com/pnnl/tesp/pull/14/files#diff-99> (3)
  *   M examples/te30/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-100> (1)
  *   D examples/te30/launch_auction.py<https://github.com/pnnl/tesp/pull/14/files#diff-101> (3)
  *   A examples/te30/launch_substation.py<https://github.com/pnnl/tesp/pull/14/files#diff-102> (3)
  *   M examples/te30/prepare_case.py<https://github.com/pnnl/tesp/pull/14/files#diff-103> (4)
  *   M examples/te30/pypower30.yaml<https://github.com/pnnl/tesp/pull/14/files#diff-104> (10)
  *   M examples/te30/run.bat<https://github.com/pnnl/tesp/pull/14/files#diff-105> (54)
  *   M examples/te30/run.sh<https://github.com/pnnl/tesp/pull/14/files#diff-106> (2)
  *   M examples/te30/run0.bat<https://github.com/pnnl/tesp/pull/14/files#diff-107> (4)
  *   M examples/te30/run0.sh<https://github.com/pnnl/tesp/pull/14/files#diff-108> (2)
  *   M examples/te30/tesp_monitor.json<https://github.com/pnnl/tesp/pull/14/files#diff-109> (10)
  *   M install/Linux/Ubuntu_build.rst<https://github.com/pnnl/tesp/pull/14/files#diff-110> (28)
  *   A install/Linux/Ubuntu_install.rst<https://github.com/pnnl/tesp/pull/14/files#diff-111> (52)
  *   M install/MacOSX/MacOSX_build.rst<https://github.com/pnnl/tesp/pull/14/files#diff-112> (28)
  *   A install/MacOSX/MacOSX_install.rst<https://github.com/pnnl/tesp/pull/14/files#diff-113> (19)
  *   M install/Windows/Windows_build.rst<https://github.com/pnnl/tesp/pull/14/files#diff-114> (7)
  *   M install/Windows/Windows_install.rst<https://github.com/pnnl/tesp/pull/14/files#diff-115> (35)
  *   A readthedocs.yml<https://github.com/pnnl/tesp/pull/14/files#diff-116> (5)
  *   M src/tesp_support/setup.py<https://github.com/pnnl/tesp/pull/14/files#diff-117> (2)
  *   M src/tesp_support/tesp_support/TMY2EPW.py<https://github.com/pnnl/tesp/pull/14/files#diff-118> (20)
  *   M src/tesp_support/tesp_support/api.py<https://github.com/pnnl/tesp/pull/14/files#diff-119> (26)
  *   M src/tesp_support/tesp_support/feederGenerator.py<https://github.com/pnnl/tesp/pull/14/files#diff-120> (343)
  *   M src/tesp_support/tesp_support/fncs.py<https://github.com/pnnl/tesp/pull/14/files#diff-121> (133)
  *   M src/tesp_support/tesp_support/fncsPYPOWER.py<https://github.com/pnnl/tesp/pull/14/files#diff-122> (54)
  *   M src/tesp_support/tesp_support/glm_dict.py<https://github.com/pnnl/tesp/pull/14/files#diff-123> (78)
  *   A src/tesp_support/tesp_support/helpers.py<https://github.com/pnnl/tesp/pull/14/files#diff-124> (247)
  *   A src/tesp_support/tesp_support/hvac.py<https://github.com/pnnl/tesp/pull/14/files#diff-125> (203)
  *   M src/tesp_support/tesp_support/precool.py<https://github.com/pnnl/tesp/pull/14/files#diff-126> (325)
  *   D src/tesp_support/tesp_support/prep_auction.py<https://github.com/pnnl/tesp/pull/14/files#diff-127> (348)
  *   M src/tesp_support/tesp_support/prep_precool.py<https://github.com/pnnl/tesp/pull/14/files#diff-128> (179)
  *   A src/tesp_support/tesp_support/prep_substation.py<https://github.com/pnnl/tesp/pull/14/files#diff-129> (390)
  *   M src/tesp_support/tesp_support/process_agents.py<https://github.com/pnnl/tesp/pull/14/files#diff-130> (23)
  *   M src/tesp_support/tesp_support/process_eplus.py<https://github.com/pnnl/tesp/pull/14/files#diff-131> (19)
  *   M src/tesp_support/tesp_support/process_gld.py<https://github.com/pnnl/tesp/pull/14/files#diff-132> (468)
  *   M src/tesp_support/tesp_support/process_houses.py<https://github.com/pnnl/tesp/pull/14/files#diff-133> (247)
  *   M src/tesp_support/tesp_support/process_inv.py<https://github.com/pnnl/tesp/pull/14/files#diff-134> (1091)
  *   M src/tesp_support/tesp_support/process_pypower.py<https://github.com/pnnl/tesp/pull/14/files#diff-135> (26)
  *   M src/tesp_support/tesp_support/process_voltages.py<https://github.com/pnnl/tesp/pull/14/files#diff-136> (25)
  *   M src/tesp_support/tesp_support/simple_auction.py<https://github.com/pnnl/tesp/pull/14/files#diff-137> (425)
  *   R src/tesp_support/tesp_support/substation.py<https://github.com/pnnl/tesp/pull/14/files#diff-138> (59)
  *   M src/tesp_support/tesp_support/tesp_case.py<https://github.com/pnnl/tesp/pull/14/files#diff-139> (176)
  *   M src/tesp_support/tesp_support/tesp_config.py<https://github.com/pnnl/tesp/pull/14/files#diff-140> (1017)
  *   M src/tesp_support/tesp_support/tesp_monitor.py<https://github.com/pnnl/tesp/pull/14/files#diff-141> (216)
  *   M src/tesp_support/tesp_support/tesp_monitor_ercot.py<https://github.com/pnnl/tesp/pull/14/files#diff-142> (119)
  *   M support/misc/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-143> (1)
  *   M tesp.xml<https://github.com/pnnl/tesp/pull/14/files#diff-144> (172)

Patch Links:

  *   https://github.com/pnnl/tesp/pull/14.patch
  *   https://github.com/pnnl/tesp/pull/14.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/pnnl/tesp/pull/14>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKCgyDjzHDDRV_4U0iPYUqudLLQrwtC8ks5vNJcrgaJpZM4a6cS->.

temcdrm commented:
Yes, I can do the merge myself after hearing from Andy and Laurentiu.  Thanks for making the review so quickly.

_________________________

Tom McDermott, Ph.D., PE, FIEEE
Chief Engineer, Systems Team
Electricity Infrastructure Group
Pacific Northwest National Laboratory
Phone: 509-375-4434
Thomas.McDermott@pnnl.gov

From: xcosmos6 <notifications@github.com>
Reply-To: pnnl/tesp <reply@reply.github.com>
Date: Thursday, February 14, 2019 at 3:58 PM
To: pnnl/tesp <tesp@noreply.github.com>
Cc: "McDermott, Thomas E" <thomas.mcdermott@pnnl.gov>, Assign <assign@noreply.github.com>
Subject: Re: [pnnl/tesp] Develop (#14)

Hi Tom,

After approve the pull request, I don't need to merge it to the master branch, right? I assume you will be doing that?

Thanks,
Heng
________________________________
From: Tom McDermott <notifications@github.com>
Sent: Wednesday, February 13, 2019 2:52 PM
To: pnnl/tesp
Cc: Subscribed
Subject: [pnnl/tesp] Develop (#14)


This would be the base version for DSO+T study

* Using the FNCS message aggregation feature. If one subscriber uses aggregation, it appears that all subscribers to the same publisher must also use aggregation.
* Refactored and documented tesp_support classes
* New installers for Windows and Linux
* ERCOT and all the examples run after refactoring
* Updates will be required for the Docker version, and possibly for the ERCOT monitor
* See docstring conventions for the Python code in tesp_support; this appears formatted on ReadTheDocs

________________________________
You can view, comment on, or merge this pull request online at:

https://github.com/pnnl/tesp/pull/14

Commit Summary

* FNCS updates
* updates for dictionary changes and FNCS message aggregation
* first file with docstrings
* for readthedocs to work
* looking for tesp_support
* Python 3.6 for ReadTheDocs
* Python 3.6 for ReadTheDocs
* class refactoring
* updating module selections
* working on fix to the tesp_monitor for te30
* testing with refactored classes
* refactored precooling classes
* precool agent estimates the ETP parameters
* follow TE30 rules for the inverter meters
* testing the sgip1 case; all simulations run but the HVAC rebound effect at 6 p.m. each day is more pronounced than before:
* fixing tesp_monitor
* reorganized the sections, added the Napoleon extension for Python docstrings
* autonumber figures and tables
* rolling the version number
* testing on Windows
* rebuild for possible fncs aggregation
* rebuild for FNCS message aggregation
* testing on Linux
* some docstrings
* installer for 0.3.0
* updating documentation
* adding docstrings
* docstrings
* docstrings for simple_auction, added Todo lists
* finished the docstrings
* rolling version number with docstrings
* updates to kill5570 for Windows 7, and python3 added to some Linux/Mac shell scripts
* installation tested for Ubuntu 18.04 LTS
* updating instructions
* updating example instructions
* draft documentation for DSO+T agents
* updating installation and build instructions
* added exterior doors to GridLAB-D dictionary

File Changes

* M README.md<https://github.com/pnnl/tesp/pull/14/files#diff-0> (28)
* A doc/Bibliography.rst<https://github.com/pnnl/tesp/pull/14/files#diff-1> (96)
* A doc/Building_TESP.rst<https://github.com/pnnl/tesp/pull/14/files#diff-2> (26)
* R doc/Code_Reference.rst<https://github.com/pnnl/tesp/pull/14/files#diff-3> (52)
* A doc/DSOT_Study.rst<https://github.com/pnnl/tesp/pull/14/files#diff-4> (63)
* A doc/Design_Reference.rst<https://github.com/pnnl/tesp/pull/14/files#diff-5> (574)
* A doc/Installing_MacOSX_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-6> (2)
* A doc/Installing_Ubuntu_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-7> (2)
* A doc/Installing_Windows_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-8> (2)
* A doc/Linux_Build_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-9> (1)
* A doc/MacOSX_Build_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-10> (1)
* A doc/Next_Version.rst<https://github.com/pnnl/tesp/pull/14/files#diff-11> (214)
* A doc/Running_Examples_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-12> (2)
* A doc/SGIP1_Example.rst<https://github.com/pnnl/tesp/pull/14/files#diff-13> (315)
* M doc/TESP.eap<https://github.com/pnnl/tesp/pull/14/files#diff-14> (0)
* D doc/TESP_DesignDoc.html<https://github.com/pnnl/tesp/pull/14/files#diff-15> (393)
* D doc/TESP_DesignDoc.rst<https://github.com/pnnl/tesp/pull/14/files#diff-16> (1376)
* A doc/TESP_Overview.rst<https://github.com/pnnl/tesp/pull/14/files#diff-17> (170)
* A doc/Using_TESP.rst<https://github.com/pnnl/tesp/pull/14/files#diff-18> (33)
* A doc/Windows_Build_Link.rst<https://github.com/pnnl/tesp/pull/14/files#diff-19> (1)
* M doc/conf.py<https://github.com/pnnl/tesp/pull/14/files#diff-20> (22)
* M doc/index.rst<https://github.com/pnnl/tesp/pull/14/files#diff-21> (13)
* M doc/media/GLDDictionary.png<https://github.com/pnnl/tesp/pull/14/files#diff-22> (0)
* M doc/media/TESPComposition.png<https://github.com/pnnl/tesp/pull/14/files#diff-23> (0)
* M doc/media/tesp_support.png<https://github.com/pnnl/tesp/pull/14/files#diff-24> (0)
* A doc/requirements.txt<https://github.com/pnnl/tesp/pull/14/files#diff-25> (5)
* M doc/tesp_support.inc<https://github.com/pnnl/tesp/pull/14/files#diff-26> (66)
* M ercot/case8/.gitignore<https://github.com/pnnl/tesp/pull/14/files#diff-27> (1)
* M ercot/case8/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-28> (1)
* M ercot/case8/plot_gld.py<https://github.com/pnnl/tesp/pull/14/files#diff-29> (4)
* R ercot/case8/prep_ercot_substation.py<https://github.com/pnnl/tesp/pull/14/files#diff-30> (14)
* M ercot/case8/prepare_case.py<https://github.com/pnnl/tesp/pull/14/files#diff-31> (18)
* M ercot/case8/pypower8.yaml<https://github.com/pnnl/tesp/pull/14/files#diff-32> (80)
* M ercot/case8/run_market.sh<https://github.com/pnnl/tesp/pull/14/files#diff-33> (16)
* M ercot/case8/run_market_debug.sh<https://github.com/pnnl/tesp/pull/14/files#diff-34> (16)
* A ercot/pdf/DSO+T_Water_Heater_Agent.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-35> (0)
* A ercot/pdf/DayAheadBidsBattery.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-36> (0)
* A ercot/pdf/EWH_11_6_2018.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-37> (0)
* A ercot/pdf/Fixed_Layers_Stratified_Water_Heater.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-38> (0)
* A ercot/pdf/MasterScriptDocumentation.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-39> (0)
* A ercot/pdf/Transformer_transactive_control.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-40> (0)
* A ercot/pdf/Water_Heater_Model_Deficiency_for_B2G.pdf<https://github.com/pnnl/tesp/pull/14/files#diff-41> (0)
* M examples/Running_Examples.rst<https://github.com/pnnl/tesp/pull/14/files#diff-42> (111)
* M examples/energyplus/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-43> (10)
* M examples/energyplus/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-44> (1)
* M examples/ieee8500/PNNLteam/bill.py<https://github.com/pnnl/tesp/pull/14/files#diff-45> (6)
* M examples/ieee8500/PNNLteam/inv30.glm<https://github.com/pnnl/tesp/pull/14/files#diff-46> (27)
* M examples/ieee8500/PNNLteam/inv8500.glm<https://github.com/pnnl/tesp/pull/14/files#diff-47> (2)
* A examples/ieee8500/PNNLteam/invti30.glm<https://github.com/pnnl/tesp/pull/14/files#diff-48> (3558)
* M examples/ieee8500/PNNLteam/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-49> (1)
* M examples/ieee8500/PNNLteam/plot_invs.py<https://github.com/pnnl/tesp/pull/14/files#diff-50> (6)
* M examples/ieee8500/PNNLteam/prepare_cases.py<https://github.com/pnnl/tesp/pull/14/files#diff-51> (4)
* M examples/ieee8500/PNNLteam/run30.sh<https://github.com/pnnl/tesp/pull/14/files#diff-52> (2)
* M examples/ieee8500/PNNLteam/run8500.sh<https://github.com/pnnl/tesp/pull/14/files#diff-53> (2)
* A examples/ieee8500/PNNLteam/runti30.sh<https://github.com/pnnl/tesp/pull/14/files#diff-54> (6)
* M examples/ieee8500/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-55> (61)
* M examples/ieee8500/clean.bat<https://github.com/pnnl/tesp/pull/14/files#diff-56> (1)
* M examples/ieee8500/clean.sh<https://github.com/pnnl/tesp/pull/14/files#diff-57> (1)
* M examples/loadshed/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-58> (4)
* M examples/loadshed/java/JNIfncs.dll<https://github.com/pnnl/tesp/pull/14/files#diff-59> (0)
* M examples/loadshed/java/libJNIfncs.jnilib<https://github.com/pnnl/tesp/pull/14/files#diff-60> (0)
* M examples/loadshed/java/libJNIfncs.so<https://github.com/pnnl/tesp/pull/14/files#diff-61> (0)
* M examples/loadshed/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-62> (1)
* M examples/loadshed/run.sh<https://github.com/pnnl/tesp/pull/14/files#diff-63> (2)
* M examples/pypower/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-64> (7)
* M examples/pypower/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-65> (1)
* M examples/pypower/runpp.sh<https://github.com/pnnl/tesp/pull/14/files#diff-66> (2)
* M examples/sgip1/.gitignore<https://github.com/pnnl/tesp/pull/14/files#diff-67> (2)
* M examples/sgip1/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-68> (16)
* M examples/sgip1/SGIP1a.glm<https://github.com/pnnl/tesp/pull/14/files#diff-69> (2)
* M examples/sgip1/SGIP1b.glm<https://github.com/pnnl/tesp/pull/14/files#diff-70> (2)
* M examples/sgip1/SGIP1c.glm<https://github.com/pnnl/tesp/pull/14/files#diff-71> (2)
* M examples/sgip1/SGIP1d.glm<https://github.com/pnnl/tesp/pull/14/files#diff-72> (2)
* M examples/sgip1/SGIP1e.glm<https://github.com/pnnl/tesp/pull/14/files#diff-73> (2)
* M examples/sgip1/prepare_cases.py<https://github.com/pnnl/tesp/pull/14/files#diff-74> (20)
* M examples/sgip1/pypower.yaml<https://github.com/pnnl/tesp/pull/14/files#diff-75> (10)
* M examples/sgip1/runSGIP1a.bat<https://github.com/pnnl/tesp/pull/14/files#diff-76> (4)
* M examples/sgip1/runSGIP1a.sh<https://github.com/pnnl/tesp/pull/14/files#diff-77> (4)
* M examples/sgip1/runSGIP1b.bat<https://github.com/pnnl/tesp/pull/14/files#diff-78> (4)
* M examples/sgip1/runSGIP1b.sh<https://github.com/pnnl/tesp/pull/14/files#diff-79> (4)
* M examples/sgip1/runSGIP1c.bat<https://github.com/pnnl/tesp/pull/14/files#diff-80> (4)
* M examples/sgip1/runSGIP1c.sh<https://github.com/pnnl/tesp/pull/14/files#diff-81> (4)
* M examples/sgip1/runSGIP1d.bat<https://github.com/pnnl/tesp/pull/14/files#diff-82> (4)
* M examples/sgip1/runSGIP1d.sh<https://github.com/pnnl/tesp/pull/14/files#diff-83> (4)
* M examples/sgip1/runSGIP1e.bat<https://github.com/pnnl/tesp/pull/14/files#diff-84> (4)
* M examples/sgip1/runSGIP1e.sh<https://github.com/pnnl/tesp/pull/14/files#diff-85> (4)
* M examples/sgip1/runSGIP1ex.bat<https://github.com/pnnl/tesp/pull/14/files#diff-86> (4)
* M examples/sgip1/runSGIP1ex.sh<https://github.com/pnnl/tesp/pull/14/files#diff-87> (5)
* M examples/sgip1/runall.sh<https://github.com/pnnl/tesp/pull/14/files#diff-88> (13)
* M examples/te30/.gitignore<https://github.com/pnnl/tesp/pull/14/files#diff-89> (3)
* M examples/te30/README.md<https://github.com/pnnl/tesp/pull/14/files#diff-90> (26)
* A examples/te30/TESP_Monitor.yaml<https://github.com/pnnl/tesp/pull/14/files#diff-91> (36)
* M examples/te30/TE_Challenge.glm<https://github.com/pnnl/tesp/pull/14/files#diff-92> (2)
* M examples/te30/clean.bat<https://github.com/pnnl/tesp/pull/14/files#diff-93> (1)
* M examples/te30/clean.sh<https://github.com/pnnl/tesp/pull/14/files#diff-94> (2)
* M examples/te30/eplus_json.yaml<https://github.com/pnnl/tesp/pull/14/files#diff-95> (2)
* D examples/te30/fncs.zpl<https://github.com/pnnl/tesp/pull/14/files#diff-96> (23)
* A examples/te30/gui.bat<https://github.com/pnnl/tesp/pull/14/files#diff-97> (3)
* A examples/te30/gui.py<https://github.com/pnnl/tesp/pull/14/files#diff-98> (6)
* A examples/te30/gui.sh<https://github.com/pnnl/tesp/pull/14/files#diff-99> (3)
* M examples/te30/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-100> (1)
* D examples/te30/launch_auction.py<https://github.com/pnnl/tesp/pull/14/files#diff-101> (3)
* A examples/te30/launch_substation.py<https://github.com/pnnl/tesp/pull/14/files#diff-102> (3)
* M examples/te30/prepare_case.py<https://github.com/pnnl/tesp/pull/14/files#diff-103> (4)
* M examples/te30/pypower30.yaml<https://github.com/pnnl/tesp/pull/14/files#diff-104> (10)
* M examples/te30/run.bat<https://github.com/pnnl/tesp/pull/14/files#diff-105> (54)
* M examples/te30/run.sh<https://github.com/pnnl/tesp/pull/14/files#diff-106> (2)
* M examples/te30/run0.bat<https://github.com/pnnl/tesp/pull/14/files#diff-107> (4)
* M examples/te30/run0.sh<https://github.com/pnnl/tesp/pull/14/files#diff-108> (2)
* M examples/te30/tesp_monitor.json<https://github.com/pnnl/tesp/pull/14/files#diff-109> (10)
* M install/Linux/Ubuntu_build.rst<https://github.com/pnnl/tesp/pull/14/files#diff-110> (28)
* A install/Linux/Ubuntu_install.rst<https://github.com/pnnl/tesp/pull/14/files#diff-111> (52)
* M install/MacOSX/MacOSX_build.rst<https://github.com/pnnl/tesp/pull/14/files#diff-112> (28)
* A install/MacOSX/MacOSX_install.rst<https://github.com/pnnl/tesp/pull/14/files#diff-113> (19)
* M install/Windows/Windows_build.rst<https://github.com/pnnl/tesp/pull/14/files#diff-114> (7)
* M install/Windows/Windows_install.rst<https://github.com/pnnl/tesp/pull/14/files#diff-115> (35)
* A readthedocs.yml<https://github.com/pnnl/tesp/pull/14/files#diff-116> (5)
* M src/tesp_support/setup.py<https://github.com/pnnl/tesp/pull/14/files#diff-117> (2)
* M src/tesp_support/tesp_support/TMY2EPW.py<https://github.com/pnnl/tesp/pull/14/files#diff-118> (20)
* M src/tesp_support/tesp_support/api.py<https://github.com/pnnl/tesp/pull/14/files#diff-119> (26)
* M src/tesp_support/tesp_support/feederGenerator.py<https://github.com/pnnl/tesp/pull/14/files#diff-120> (343)
* M src/tesp_support/tesp_support/fncs.py<https://github.com/pnnl/tesp/pull/14/files#diff-121> (133)
* M src/tesp_support/tesp_support/fncsPYPOWER.py<https://github.com/pnnl/tesp/pull/14/files#diff-122> (54)
* M src/tesp_support/tesp_support/glm_dict.py<https://github.com/pnnl/tesp/pull/14/files#diff-123> (78)
* A src/tesp_support/tesp_support/helpers.py<https://github.com/pnnl/tesp/pull/14/files#diff-124> (247)
* A src/tesp_support/tesp_support/hvac.py<https://github.com/pnnl/tesp/pull/14/files#diff-125> (203)
* M src/tesp_support/tesp_support/precool.py<https://github.com/pnnl/tesp/pull/14/files#diff-126> (325)
* D src/tesp_support/tesp_support/prep_auction.py<https://github.com/pnnl/tesp/pull/14/files#diff-127> (348)
* M src/tesp_support/tesp_support/prep_precool.py<https://github.com/pnnl/tesp/pull/14/files#diff-128> (179)
* A src/tesp_support/tesp_support/prep_substation.py<https://github.com/pnnl/tesp/pull/14/files#diff-129> (390)
* M src/tesp_support/tesp_support/process_agents.py<https://github.com/pnnl/tesp/pull/14/files#diff-130> (23)
* M src/tesp_support/tesp_support/process_eplus.py<https://github.com/pnnl/tesp/pull/14/files#diff-131> (19)
* M src/tesp_support/tesp_support/process_gld.py<https://github.com/pnnl/tesp/pull/14/files#diff-132> (468)
* M src/tesp_support/tesp_support/process_houses.py<https://github.com/pnnl/tesp/pull/14/files#diff-133> (247)
* M src/tesp_support/tesp_support/process_inv.py<https://github.com/pnnl/tesp/pull/14/files#diff-134> (1091)
* M src/tesp_support/tesp_support/process_pypower.py<https://github.com/pnnl/tesp/pull/14/files#diff-135> (26)
* M src/tesp_support/tesp_support/process_voltages.py<https://github.com/pnnl/tesp/pull/14/files#diff-136> (25)
* M src/tesp_support/tesp_support/simple_auction.py<https://github.com/pnnl/tesp/pull/14/files#diff-137> (425)
* R src/tesp_support/tesp_support/substation.py<https://github.com/pnnl/tesp/pull/14/files#diff-138> (59)
* M src/tesp_support/tesp_support/tesp_case.py<https://github.com/pnnl/tesp/pull/14/files#diff-139> (176)
* M src/tesp_support/tesp_support/tesp_config.py<https://github.com/pnnl/tesp/pull/14/files#diff-140> (1017)
* M src/tesp_support/tesp_support/tesp_monitor.py<https://github.com/pnnl/tesp/pull/14/files#diff-141> (216)
* M src/tesp_support/tesp_support/tesp_monitor_ercot.py<https://github.com/pnnl/tesp/pull/14/files#diff-142> (119)
* M support/misc/kill5570.bat<https://github.com/pnnl/tesp/pull/14/files#diff-143> (1)
* M tesp.xml<https://github.com/pnnl/tesp/pull/14/files#diff-144> (172)

Patch Links:

* https://github.com/pnnl/tesp/pull/14.patch
* https://github.com/pnnl/tesp/pull/14.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/pnnl/tesp/pull/14>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AKCgyDjzHDDRV_4U0iPYUqudLLQrwtC8ks5vNJcrgaJpZM4a6cS->.


—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://github.com/pnnl/tesp/pull/14#issuecomment-463851579>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AIFMT-GohoWTK3q-DllSM-77KtQH4puLks5vNffhgaJpZM4a6cS->.
